### PR TITLE
Death squad changes

### PR DESCRIPTION
**Pull request by KEEYNy**
## About the PR

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added Death Squad jumpsuit
- tweak: Death Squad equipment now includes jumpsuit and Chef Engineer magboots

### CODE REVIEW

**Comment by KEEYNy** [comment]
image
image
image

**Comment by brainfood1183** [comment]
Any chance you could add a monkey version? monkeys can now wear jumpsuits and should have a monkey jumpsuit sprite. You can use one of the existing monkey jumpsuit sprites as guide.

**Comment by KEEYNy** [comment]
Monkey death squad, sounds impossible

**Comment by deltanedas** [comment]
go beyond impossible, kick reason to the curb

**Comment by brainfood1183** [comment]
> Monkey death squad, sounds impossible

Nothing is impossible with admeme

**Comment by LordEclipse** [comment]
image

**Comment by brainfood1183** [comment]
> image

which jumpsuit is that?

**Comment by deltanedas** [comment]
its the hardsuit not jumpsuit

**Comment by LordEclipse** [comment]
> > image
> 
> which jumpsuit is that?

Deadsquad set outfit currently gives them a black jumpsuit

**Comment by KEEYNy** [comment]
Black jumpsuit like greytide?)))
My jumpsuit also has protection

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+    sprite: Clothing/Uniforms/Jumpsuit/deadsquad.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpsuit/deadsquad.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.8
+        Slash: 0.8
+        Piercing: 0.8
+        Heat: 0.8
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:17]: Jumpsuits shouldn't have armour.

File: Resources/Textures/Clothing/Uniforms/Jumpsuit/deadsquad.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Sprited by KEEYNy (Github) for Space Station 14. contacts: https://github.com/KEEYNy  discord: KEEYN#3805",
```
Comment (metalgearsloth) [inline Resources/Textures/Clothing/Uniforms/Jumpsuit/deadsquad.rsi/meta.json:4]: Sprite looks derivative?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml (hunk 1 above)
Comment (KEEYNy) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:17]: ERT have, why should not death squad ?

Comment (KEEYNy) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:17]: ERT has armor, but the squadron shouldn't?  this is weird.  From the point of view of balance, the squadron should not die at all, and even these 20% will not change anything

Comment (deltanedas) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:17]: also hos turtleneck has a bit of armour too

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 3]
```diff
@@ -1,3 +1,21 @@
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitDeadSquad
+  name: Dead Squad uniform
```
Comment (lapatison) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:4]: Should be `deathsquad`, entity ID too

Comment (deltanedas) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:4]: its deathsquad not Dead Squad

Comment (KEEYNy) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:4]: image

Comment (deltanedas) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:4]: so fix the typo pro dont continue it
also you ignore ClothingOuterHardsuitDeathsquad
and the comment above it

**Comment by Maksim22848** [comment]
Сам ты гей

**Pull request merged**